packages/itwin/tree-widget/README.md

```diff
@@ -331,35 +418,26 @@ In general, the component is expected to be used by advanced users to inspect co
 
 Typical usage:
 
+  <!-- [[include: [Presentation.Tree-widget.Imodel-content-tree-example-imports, Presentation.Tree-widget.Imodel-content-tree-example], tsx]] -->
+  <!-- BEGIN EXTRACTION -->
+
 ```tsx
```

Comment:
any idea why the code block is not indented the same amount as the `<!-- [[include: ... ]] -->` comment?

Reply:
Removed indentation for the include part, not sure what you mean by this?

Reply:
The idea is that our script should indent the code block as much as the `<!-- [[include: ... ]] -->` is indented. This is needed for cases when code blocks need to be created under list items, for example.